ai2thor/wsgi_server.py

```diff
@@ -27,11 +33,16 @@
 
 werkzeug.serving.WSGIRequestHandler.protocol_version = "HTTP/1.1"
 
+
 # get with timeout to allow quit
-def queue_get(que, unity_proc=None):
+def queue_get(que, unity_proc=None, timeout: Optional[float] = 100.0):
     res = None
     attempts = 0
-    max_attempts = 200
+    max_attempts = (
+        float("inf")
+        if timeout is None or timeout == float("inf") else
+        max(int(math.ceil(timeout / 0.5)), 1)
```

Comment:
A line is a bit hard to read, any reason for ` / 0.5` vs ` * 2`, making it floating point?

Reply:
I've made this a bit clearer, the idea is that the 0.5 references the 0.5 used by the `que.get(...)` call a few lines layer.